### PR TITLE
Pages/direct uploads doc

### PR DESCRIPTION
**Pull request by Ekwuno**

### CODE REVIEW

File: content/pages/platform/direct-upload.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+pcx-content-type: concept
+title: Direct Upload
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
title: Direct Uploads
```

File: content/pages/platform/direct-upload.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+---
+
+# Direct Upload
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
# Direct Uploads
```

File: content/pages/platform/direct-upload.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+With Direct Uploads, you can bring your pre-built assets right to Pages. By using your own CI tooling to handle the build, deploy your website's assets right to the Cloudflare edge.
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
With Direct Uploads, you can bring your pre-built assets right to Pages. By using your own continuous integration (CI) tooling to handle the build, deploy your website's assets right to the Cloudflare edge network.
```

File: content/pages/platform/direct-upload.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+## Methods to uploading
+
+Once you have your pre-built assets ready, there are two ways to begin uploading: 
+
+1. Using Wrangler CLI 
+2. Dragging and dropping
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
1. Using [Wrangler CLI](https://developers.cloudflare.com/workers/wrangler/).
2. Dragging and dropping.
```

File: content/pages/platform/direct-upload.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+## Methods to uploading
+
+Once you have your pre-built assets ready, there are two ways to begin uploading: 
+
+1. Using Wrangler CLI 
+2. Dragging and dropping 
+
+{{<Aside type= "note">}}
+  
+Within a project, you can switch between creating deployments with either method. However, at this time, you cannot additionally push changes through your standard git-integration within the same project._
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
Within a project, you can switch between creating deployments with either method. However, you cannot push additional changes through your standard git integration within the same project if that repository is already using Direct Uploads.
```

File: content/pages/platform/direct-upload.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+
+{{</Aside>}}
+
+## Supported file types:
+
+* Wrangler: a single folder of assets(e.g. HTML, CSS, JS, PNG, SVG)
+* Drag and drop: zip or folder of assets(e.g. HTML, CSS, JS, PNG, SVG) 
+
+
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md:28]:
```suggestion
```

File: content/pages/platform/direct-upload.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+{{<Aside type= "note">}}
+  
+Within a project, you can switch between creating deployments with either method. However, at this time, you cannot additionally push changes through your standard git-integration within the same project._
+
+{{</Aside>}}
+
+## Supported file types:
+
+* Wrangler: a single folder of assets(e.g. HTML, CSS, JS, PNG, SVG)
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
* Wrangler: A single folder of assets(for example, HTML, CSS, JS, PNG, SVG).
```

File: content/pages/platform/direct-upload.md [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+* Drag and drop: zip or folder of assets(e.g. HTML, CSS, JS, PNG, SVG)
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
* Drag and drop: Zip or folder of assets(for example, HTML, CSS, JS, PNG, SVG).
```

File: content/pages/platform/direct-upload.md [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+2. Dragging and dropping 
+
+{{<Aside type= "note">}}
+  
+Within a project, you can switch between creating deployments with either method. However, at this time, you cannot additionally push changes through your standard git-integration within the same project._
+
+{{</Aside>}}
+
+## Supported file types:
+
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion

Below is _____:
```
Explain what we are listing out.

File: content/pages/platform/direct-upload.md [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+* Wrangler: a single folder of assets(e.g. HTML, CSS, JS, PNG, SVG)
+* Drag and drop: zip or folder of assets(e.g. HTML, CSS, JS, PNG, SVG) 
+
+
+## Wrangler CLI 
+
+### Setting up Wrangler:
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
### Set up Wrangler: 
```

File: content/pages/platform/direct-upload.md [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+
+* Wrangler: a single folder of assets(e.g. HTML, CSS, JS, PNG, SVG)
+* Drag and drop: zip or folder of assets(e.g. HTML, CSS, JS, PNG, SVG) 
+
+
+## Wrangler CLI 
+
+### Setting up Wrangler: 
+
+To begin, be sure to [install the latest version](https://developers.cloudflare.com/workers/cli-wrangler/install-update/) of Wrangler and [set up Wrangler](https://developers.cloudflare.com/workers/cli-wrangler/authentication/) to work with your Cloudflare user. Please note that Pages integration with Wrangler relies on Wrangler 2.
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
To begin, [install the latest version](https://developers.cloudflare.com/workers/cli-wrangler/install-update/) of Wrangler and [set up Wrangler](https://developers.cloudflare.com/workers/cli-wrangler/authentication/). Note that Pages integration with Wrangler relies on Wrangler 2.
```

File: content/pages/platform/direct-upload.md [hunk 12]
```diff
@@ -0,0 +1,102 @@
...
+
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md:32]:
```suggestion
```

File: content/pages/platform/direct-upload.md [hunk 13]
```diff
@@ -0,0 +1,102 @@
...
+
+#### Deploying your first project:
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
#### Deploy your project with Wrangler
```

File: content/pages/platform/direct-upload.md [hunk 14]
```diff
@@ -0,0 +1,102 @@
...
+## Wrangler CLI 
+
+### Setting up Wrangler: 
+
+To begin, be sure to [install the latest version](https://developers.cloudflare.com/workers/cli-wrangler/install-update/) of Wrangler and [set up Wrangler](https://developers.cloudflare.com/workers/cli-wrangler/authentication/) to work with your Cloudflare user. Please note that Pages integration with Wrangler relies on Wrangler 2.
+
+
+#### Deploying your first project: 
+
+Execute the following Wrangler command to begin project creation:
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
Run the following Wrangler command to create a project: 
```

File: content/pages/platform/direct-upload.md [hunk 15]
```diff
@@ -0,0 +1,102 @@
...
+
+
+#### Deploying your first project: 
+
+Execute the following Wrangler command to begin project creation: 
+
+```sh
+wrangler pages publish <project directory>
+```
+
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md:42]:
```suggestion
```

File: content/pages/platform/direct-upload.md [hunk 16]
```diff
@@ -0,0 +1,102 @@
...
+
+Once executed, you will be prompted to choose whether you’d like to publish assets for an existing project or if you’d like to create a new one. To begin a new project, indicate so with the options provided, continue to name your project, and deploy. Subsequent deployments will re-use these values (saved in your `node_modules/.cache/wrangler` folder).
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
After running `wrangler pages publish`, you will be prompted to choose whether you would like to publish assets for an existing project or if you would like to create a new one. To begin a new project, indicate so with the options provided, continue to name your project, and deploy. Subsequent deployments will re-use these values (saved in your `node_modules/.cache/wrangler` folder).
```

Comment (deadlypants1973) [reply content/pages/platform/direct-upload.md]: What are the options provided in "indicate so with the options provided"?

File: content/pages/platform/direct-upload.md [hunk 17]
```diff
@@ -0,0 +1,102 @@
...
+
+After your first deployment, in the Pages dashboard, you can navigate to your newly created project to access deployment details, including its shareable / unique preview URL. 
+
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
```

File: content/pages/platform/direct-upload.md [hunk 18]
```diff
@@ -0,0 +1,102 @@
...
+Execute the following Wrangler command to begin project creation: 
+
+```sh
+wrangler pages publish <project directory>
+```
+
+
+Once executed, you will be prompted to choose whether you’d like to publish assets for an existing project or if you’d like to create a new one. To begin a new project, indicate so with the options provided, continue to name your project, and deploy. Subsequent deployments will re-use these values (saved in your `node_modules/.cache/wrangler` folder).
+
+After your first deployment, in the Pages dashboard, you can navigate to your newly created project to access deployment details, including its shareable / unique preview URL.
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
After you deploy your Project, go to your newly created Pages project in the Cloudflare dashboard to access deployment details, including its shareable and unique preview URL. 
```

File: content/pages/platform/direct-upload.md [hunk 19]
```diff
@@ -0,0 +1,102 @@
...
+wrangler pages publish <project directory>
+```
+
+
+Once executed, you will be prompted to choose whether you’d like to publish assets for an existing project or if you’d like to create a new one. To begin a new project, indicate so with the options provided, continue to name your project, and deploy. Subsequent deployments will re-use these values (saved in your `node_modules/.cache/wrangler` folder).
+
+After your first deployment, in the Pages dashboard, you can navigate to your newly created project to access deployment details, including its shareable / unique preview URL. 
+
+
+#### Creating a new deployment:
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
#### Create a new deployment
```

File: content/pages/platform/direct-upload.md [hunk 20]
```diff
@@ -0,0 +1,102 @@
...
+
+Once you’ve deployed your project, you can continue to add new deployments to that project. Deployments will be available at the following convention `<deployment>.<project name>.pages.dev`. 
+
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
After you have deployed your project, you can continue to add new deployments to that project. Deployments will be available at the following convention: `<DEPLOYMENT>.<PROJECT_NAME>.pages.dev`. 

```

File: content/pages/platform/direct-upload.md [hunk 21]
```diff
@@ -0,0 +1,102 @@
...
+{{<Aside type= "note">}}
+
+If you are in a git workspace, Wrangler will automatically pull the branch information for you. Otherwise, you will be prompted to choose your branch (which will then determine if the deployment is production or preview).
+
+{{</Aside>}}
+
+#### Other useful commands
+
+If you’d like to use Wrangler to obtain a list of all available projects for direct upload, you can use:
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
If you would like to use Wrangler to obtain a list of all available projects for direct upload, use:
```

File: content/pages/platform/direct-upload.md [hunk 22]
```diff
@@ -0,0 +1,102 @@
...
+
+```sh
+wrangler pages project list
+```
+
+Additionally, if you would like to use Wrangler to obtain a list of all unique preview URLs for a particular project, you can use:
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
Additionally, if you would like to use Wrangler to obtain a list of all unique preview URLs for a particular project, use:
```

File: content/pages/platform/direct-upload.md [hunk 23]
```diff
@@ -0,0 +1,102 @@
...
+
+```sh
+wrangler pages deployment list
+```
+
+For step by step directions on how to use Wrangler and continuous integration tools like GitHub Actions, Circle CI and Travis CI together for continuous deployment, check out our [tutorial](/pages/how-to/)
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
For step-by-step directions on how to use Wrangler and continuous integration tools like GitHub Actions, Circle CI and Travis CI together for continuous deployment, refer to [Pages How to](/pages/how-to/)
```

File: content/pages/platform/direct-upload.md [hunk 24]
```diff
@@ -0,0 +1,102 @@
...
+
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
```

File: content/pages/platform/direct-upload.md [hunk 25]
```diff
@@ -0,0 +1,102 @@
...
+
+## Drag and drop
+
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md:78]:
```suggestion
```

File: content/pages/platform/direct-upload.md [hunk 26]
```diff
@@ -0,0 +1,102 @@
...
+
+#### Deploying your first project
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
#### Deploy your project with drag and drop
```

File: content/pages/platform/direct-upload.md [hunk 27]
```diff
@@ -0,0 +1,102 @@
...
+
+To begin, on the **Create a Project** page, select **Upload Assets** and enter your project name in the provided field. Your project will be served from `<project name>.pages.dev`. Next drag and drop your build output directory into the uploading frame. Once your files have been successfully uploaded, click **Save and Deploy** and continue to your newly deployed project. 
+
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
```

File: content/pages/platform/direct-upload.md [hunk 28]
```diff
@@ -0,0 +1,102 @@
...
+For step by step directions on how to use Wrangler and continuous integration tools like GitHub Actions, Circle CI and Travis CI together for continuous deployment, check out our [tutorial](/pages/how-to/)
+
+
+## Drag and drop
+
+
+#### Deploying your first project
+
+To begin, on the **Create a Project** page, select **Upload Assets** and enter your project name in the provided field. Your project will be served from `<project name>.pages.dev`. Next drag and drop your build output directory into the uploading frame. Once your files have been successfully uploaded, click **Save and Deploy** and continue to your newly deployed project.
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
To begin ________, go to **Pages** in the Cloudflare dashboard > **Create a Project** > select **Upload Assets** and enter your project name in the provided field. Your project will be served from `<PROJECT_NAME>.pages.dev`. Next drag and drop your build output directory into the uploading frame. After your files have been successfully uploaded, select **Save and Deploy** and continue to your newly deployed project. 
```

Comment (deadlypants1973) [reply content/pages/platform/direct-upload.md]: To begin doing what?

File: content/pages/platform/direct-upload.md [hunk 29]
```diff
@@ -0,0 +1,102 @@
...
+
+#### Deploying your first project
+
+To begin, on the **Create a Project** page, select **Upload Assets** and enter your project name in the provided field. Your project will be served from `<project name>.pages.dev`. Next drag and drop your build output directory into the uploading frame. Once your files have been successfully uploaded, click **Save and Deploy** and continue to your newly deployed project. 
+
+
+#### Creating a new deployment
+
+Once you have your project created, select **Create a new deployment** to begin a new version of your site. Next, choose whether your new deployment will be made to your production or preview environment. If choosing preview, you can create a new deployment target or enter an existing one. Deployment targets allow you to access all changes at one preview subdomain with the following convention `<deployment target>.<project name>.pages.dev`. 
+
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md:86]:
```suggestion
```

File: content/pages/platform/direct-upload.md [hunk 30]
```diff
@@ -0,0 +1,102 @@
...
+
+#### Deploying your first project
+
+To begin, on the **Create a Project** page, select **Upload Assets** and enter your project name in the provided field. Your project will be served from `<project name>.pages.dev`. Next drag and drop your build output directory into the uploading frame. Once your files have been successfully uploaded, click **Save and Deploy** and continue to your newly deployed project. 
+
+
+#### Creating a new deployment
+
+Once you have your project created, select **Create a new deployment** to begin a new version of your site. Next, choose whether your new deployment will be made to your production or preview environment. If choosing preview, you can create a new deployment target or enter an existing one. Deployment targets allow you to access all changes at one preview subdomain with the following convention `<deployment target>.<project name>.pages.dev`.
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
After you have your project created, select **Create a new deployment** to begin a new version of your site. Next, choose whether your new deployment will be made to your production or preview environment. If choosing preview, you can create a new deployment target or enter an existing one. Deployment targets allow you to access all changes at one preview subdomain with the following convention `<DEPLOYMENT_TARGET>.<PROJECT_NAME>.pages.dev`. 
```

File: content/pages/platform/direct-upload.md [hunk 31]
```diff
@@ -0,0 +1,102 @@
...
+
+## Drag and drop
+
+
+#### Deploying your first project
+
+To begin, on the **Create a Project** page, select **Upload Assets** and enter your project name in the provided field. Your project will be served from `<project name>.pages.dev`. Next drag and drop your build output directory into the uploading frame. Once your files have been successfully uploaded, click **Save and Deploy** and continue to your newly deployed project. 
+
+
+#### Creating a new deployment
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
#### Create a new deployment
```

File: content/pages/platform/direct-upload.md [hunk 32]
```diff
@@ -0,0 +1,102 @@
...
+
+Once you have your project created, select **Create a new deployment** to begin a new version of your site. Next, choose whether your new deployment will be made to your production or preview environment. If choosing preview, you can create a new deployment target or enter an existing one. Deployment targets allow you to access all changes at one preview subdomain with the following convention `<deployment target>.<project name>.pages.dev`. 
+
+
+## Troubleshooting
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
## Troubleshoot
```

File: content/pages/platform/direct-upload.md [hunk 33]
```diff
@@ -0,0 +1,102 @@
...
+
+When uploading your assets, if an asset exceeds the standard Pages limits of 25 MiB or if the total number of assets exceeds 20k files, you will not be able to deploy your project.
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
When uploading your assets, if an asset exceeds the standard Pages [limits](/pages/platform/limits/#files) of 25 MiB or if the total number of assets exceeds 20,000 files, you will not be able to deploy your project. 
```

File: content/pages/platform/direct-upload.md [hunk 34]
```diff
@@ -0,0 +1,102 @@
...
+#### Creating a new deployment
+
+Once you have your project created, select **Create a new deployment** to begin a new version of your site. Next, choose whether your new deployment will be made to your production or preview environment. If choosing preview, you can create a new deployment target or enter an existing one. Deployment targets allow you to access all changes at one preview subdomain with the following convention `<deployment target>.<project name>.pages.dev`. 
+
+
+## Troubleshooting
+
+When uploading your assets, if an asset exceeds the standard Pages limits of 25 MiB or if the total number of assets exceeds 20k files, you will not be able to deploy your project. 
+
+If using Wrangler, you will see a `Error: Pages only supports files up to 25 MiB in size. <file> is <size> in size` error for asset size exceedance and **"Error: Pages only supports up to 20,000 files in a deployment at the moment. Try a smaller project”** for asset number exceedance.
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
If using Wrangler, you will see a `Error: Pages only supports files up to 25 MiB in size. <FILE> is <SIZE> in size` error for asset size exceedance and **"Error: Pages only supports up to 20,000 files in a deployment at the moment. Try a smaller project”** for asset number exceedance. 
```

File: content/pages/platform/direct-upload.md [hunk 35]
```diff
@@ -0,0 +1,102 @@
...
+
+
+## Troubleshooting
+
+When uploading your assets, if an asset exceeds the standard Pages limits of 25 MiB or if the total number of assets exceeds 20k files, you will not be able to deploy your project. 
+
+If using Wrangler, you will see a `Error: Pages only supports files up to 25 MiB in size. <file> is <size> in size` error for asset size exceedance and **"Error: Pages only supports up to 20,000 files in a deployment at the moment. Try a smaller project”** for asset number exceedance. 
+
+If using the drag and drop method, a red warning symbol will appear next to an asset if too large and thus unsuccessfully uploaded. In this case, you may choose to delete that asset but you cannot replace it. To do so, you must reupload the entire project. If you exceed the file number limit, you will receive a {write error message here} error.
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
If using the drag and drop method, a red warning symbol will appear next to an asset if the asset is too large and thus unsuccessfully uploaded. In this case, you may choose to delete that asset but you cannot replace it. To do so, you must reupload the entire project. If you exceed the file number limit, you will receive a `{write error message here}` error. 
```
This needs to updated with error message.

File: content/pages/platform/direct-upload.md (hunk 5 above)
Comment (Ekwuno) [reply content/pages/platform/direct-upload.md]: At this time you can't push subsequent changes to the same repo with git. If the repo is already using Direct uploads.

File: content/pages/platform/direct-upload.md [hunk 36]
```diff
@@ -0,0 +1,95 @@
...
+
+# Direct Uploads
+
+With Direct Uploads, you can bring your pre-built assets right to Pages. By using your own continuous integration (CI) tooling to handle the build, deploy your website's assets right to the Cloudflare edge network.
+
+## Methods to uploading
+
+Once you have your pre-built assets ready, there are two ways to begin uploading: 
+
+1. Using [Wrangler CLI](https://developers.cloudflare.com/workers/wrangler/).
```
Comment (Ekwuno) [inline content/pages/platform/direct-upload.md]:
```suggestion
1. Using [Wrangler CLI](/workers/wrangler/).
```

File: content/pages/platform/direct-upload.md [hunk 37]
```diff
@@ -0,0 +1,95 @@
...
+2. Dragging and dropping.
+
+{{<Aside type= "note">}}
+  
+Within a project, you can switch between creating deployments with either method. However, at this time, you cannot additionally push changes through your standard git-integration within the same project._
+
+{{</Aside>}}
+
+## Supported file types:
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
## Supported file types
```

File: content/pages/platform/direct-upload.md (hunk 5 above)
Comment (deadlypants1973) [reply content/pages/platform/direct-upload.md]: Updated.

File: content/pages/platform/direct-upload.md [hunk 38]
```diff
@@ -0,0 +1,95 @@
...
+
+Below is the supported file types for each Direct Upload options:
+* Wrangler: A single folder of assets(for example, HTML, CSS, JS, PNG, SVG).
+* Drag and drop: Zip or folder of assets(for example, HTML, CSS, JS, PNG, SVG).
+
+## Wrangler CLI 
+
+### Set up Wrangler:
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
### Set up Wrangler
```

File: content/pages/platform/direct-upload.md [hunk 39]
```diff
@@ -0,0 +1,95 @@
...
+
+#### Deploy your project with Wrangler
+
+Run the following Wrangler command to create a project: 
+
+```sh
+wrangler pages publish <project directory>
+```
+
+After running `wrangler pages publish`, you will be prompted to choose whether you would like to publish assets for an existing project or if you would like to create a new one. To begin a new project, indicate so with the options provided, continue to name your project, and deploy. Subsequent deployments will re-use these values (saved in your `node_modules/.cache/wrangler` folder).
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
After running `wrangler pages publish`, you will be prompted to choose whether you would like to publish assets for an existing project or if you would like to create a new one. To begin a new project, select create a new project, continue to name your project, and deploy. Subsequent deployments will reuse these values (saved in your `node_modules/.cache/wrangler` folder).
```
Style guide thing but reuse and re-use are both correct, in the docs, we opt for no dash so `reuse`.

File: content/pages/platform/direct-upload.md [hunk 40]
```diff
@@ -0,0 +1,95 @@
...
+
+After you deploy your Project, go to your newly created Pages project in the Cloudflare dashboard to access deployment details, including its shareable and unique preview URL. 
+
+#### Create a new deployment
+
+After you have deployed your project, you can continue to add new deployments to that project. Deployments will be available at the following convention: `<DEPLOYMENT>.<PROJECT_NAME>.pages.dev`. 
+
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md:50]:
```suggestion
```

File: content/pages/platform/direct-upload.md [hunk 41]
```diff
@@ -0,0 +1,95 @@
...
+wrangler pages deployment list
+```
+
+For step-by-step directions on how to use Wrangler and continuous integration tools like GitHub Actions, Circle CI and Travis CI together for continuous deployment, refer to [Pages How to](/pages/how-to/)
+
+## Drag and drop
+
+#### Deploy your project with drag and drop
+
+To begin the deployment process, on the **Create a Project** page, select **Upload Assets** and enter your project name in the provided field. Your project will be served from `<project name>.pages.dev`. Next drag and drop your build output directory into the uploading frame. Once your files have been successfully uploaded, click **Save and Deploy** and continue to your newly deployed project.
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
To begin the deployment process, on the **Create a Project** page, select **Upload Assets** and enter your project name in the provided field. Your project will be served from `<PROJECT_NAME>.pages.dev`. Next drag and drop your build output directory into the uploading frame. Once your files have been successfully uploaded, select **Save and Deploy** and continue to your newly deployed project. 
```

File: content/pages/platform/direct-upload.md (hunk 39 above)
Comment (deadlypants1973) [reply content/pages/platform/direct-upload.md]: What do you need to select to create a new project?

File: content/pages/platform/direct-upload.md [hunk 42]
```diff
@@ -0,0 +1,95 @@
...
+
+After you deploy your Project, go to your newly created Pages project in the Cloudflare dashboard to access deployment details, including its shareable and unique preview URL. 
+
+#### Create a new deployment
+
+After you have deployed your project, you can continue to add new deployments to that project. Deployments will be available at the following convention: `<DEPLOYMENT>.<PROJECT_NAME>.pages.dev`. 
+
+
+```sh
+wrangler pages publish <directory> --branch=[branch]
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
wrangler pages publish <DIRECTORY> --branch=[BRANCH]
```

Comment (deadlypants1973) [reply content/pages/platform/direct-upload.md]: Is [branch] an input value? If so, all caps.

File: content/pages/platform/direct-upload.md [hunk 43]
```diff
@@ -0,0 +1,94 @@
...
+
+# Direct Uploads
+
+With Direct Uploads, you can bring your pre-built assets right to Pages. By using your own continuous integration (CI) tooling to handle the build, deploy your website's assets right to the Cloudflare edge network.
+
+## Methods to uploading
+
+Once you have your pre-built assets ready, there are two ways to begin uploading: 
+
+1. Using [Wrangler CLI](/workers/wrangler/).
```
Comment (Ekwuno) [inline content/pages/platform/direct-upload.md]:
```suggestion
1. Using [Wrangler CLI](/workers/wrangler/get-started/).
```

File: content/pages/platform/direct-upload.md [hunk 44]
```diff
@@ -0,0 +1,94 @@
...
+
+# Direct Uploads
+
+With Direct Uploads, you can bring your pre-built assets right to Pages. By using your own continuous integration (CI) tooling to handle the build, deploy your website's assets right to the Cloudflare edge network.
+
+## Methods to uploading
+
+Once you have your pre-built assets ready, there are two ways to begin uploading: 
+
+1. Using [Wrangler CLI](/workers/wrangler/get-started/).
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]:
```suggestion
1. Using [Wrangler CLI](https://developers.cloudflare.com/workers/wrangler/).
```

File: content/pages/platform/direct-upload.md [hunk 45]
```diff
@@ -0,0 +1,94 @@
...
+
+## Troubleshoot
+
+When uploading your assets, if an asset exceeds the standard Pages [limits](/pages/platform/limits/#files) of 25 MiB or if the total number of assets exceeds 20,000 files, you will not be able to deploy your project. 
+
+If using Wrangler, you will see a `Error: Pages only supports files up to 25 MiB in size. <FILE> is <SIZE> in size` error for asset size exceedance and **"Error: Pages only supports up to 20,000 files in a deployment at the moment. Try a smaller project”** for asset number exceedance. 
+
+If using the drag and drop method, a red warning symbol will appear next to an asset if the asset is too large and thus unsuccessfully uploaded. In this case, you may choose to delete that asset but you cannot replace it. To do so, you must reupload the entire project. If you exceed the file number limit, you will receive a `{write error message here}` error.
```
Comment (deadlypants1973) [inline content/pages/platform/direct-upload.md]: this needs an error message

Comment (Ekwuno) [reply content/pages/platform/direct-upload.md]: Updates

File: content/pages/platform/direct-upload.md [hunk 46]
```diff
@@ -0,0 +1,93 @@
...
+
+Below is the supported file types for each Direct Upload options:
+* Wrangler: A single folder of assets(for example, HTML, CSS, JS, PNG, SVG).
+* Drag and drop: Zip or folder of assets(for example, HTML, CSS, JS, PNG, SVG).
+
+## Wrangler CLI 
+
+### Set up Wrangler
+
+To begin, [install the latest version](https://developers.cloudflare.com/workers/cli-wrangler/install-update/) of Wrangler and [set up Wrangler](https://developers.cloudflare.com/workers/cli-wrangler/authentication/). Note that Pages integration with Wrangler relies on Wrangler 2.
```
Comment (Ekwuno) [inline content/pages/platform/direct-upload.md]:
```suggestion
To begin, [install the latest version](/workers/cli-wrangler/install-update/) of Wrangler and [set up Wrangler](/workers/cli-wrangler/authentication/). Note that Pages integration with Wrangler relies on Wrangler 2.
```

File: content/pages/platform/direct-upload.md [hunk 47]
```diff
@@ -0,0 +1,93 @@
...
+
+# Direct Uploads
+
+With Direct Uploads, you can bring your pre-built assets right to Pages. By using your own continuous integration (CI) tooling to handle the build, deploy your website's assets right to the Cloudflare edge network.
+
+## Methods to uploading
+
+Once you have your pre-built assets ready, there are two ways to begin uploading: 
+
+1. Using [Wrangler CLI](/workers/wrangler/get-started/).
```
Comment (Ekwuno) [inline content/pages/platform/direct-upload.md]:
```suggestion
1. Using [Wrangler CLI](/workers/cli-wrangler//).
```

**Pull request merged**